framework/validators/CompareValidator.php

```diff
@@ -78,43 +78,33 @@ class CompareValidator extends Validator
      */
     public $message;
 
+    /**
+     * @var array pairs of `operator` => `message`. When [[$message]] is not set
+     * the translated `message will be used.
+     */
+    private $operatorMessages = [
+        '==' => '{attribute} must be equal to "{compareValueOrAttribute}".',
```

Comment:
this is not really flexible. I would expect this to be done with protected method getDefaultMessages()